packages/sanity/src/core/studio/tasks/components/TasksSidebar/TasksSidebar.tsx

```diff
@@ -29,9 +36,98 @@ export function TasksStudioSidebar() {
         <motion.div variants={VARIANTS} transition={TRANSITION} initial="hidden" animate="visible">
           <SidebarRoot borderLeft height="fill" marginLeft={1}>
             <TasksSidebarHeader />
+            {isLoading ? <Spinner /> : <TasksList items={data ?? []} />}
           </SidebarRoot>
         </motion.div>
       )}
     </AnimatePresence>
   )
 }
+
+function TasksList({items}: {items: TaskDocument[]}) {
+  const currentUser = useCurrentUser()
+  const tabs = useMemo(() => {
```

Comment:
Maybe this tabs could me moved outisde of the component to avoid the useMemo since it will be stable? Wdyt?